scripts/convert_yaml_to_sdf.py

```diff
@@ -289,6 +294,8 @@ def main(model_name, recursive=False):
         file_type = "world"
     else:
         file_type = "model"
+        if recursive:
+            print("recursive doesn't apply to 'model' objects, only to 'world' objects.")
```

Comment:
Does this not cause this message to be printed for every model that is included in a world? It doesn't really hurt to get a prompt about this fact, but it's not really necessary.

Reply:
Yes indeed. I will remove it.